structuredefinitions/UKCore-Observation-VitalSigns-RespirationRate.xml

```diff
@@ -17,7 +17,7 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="Defines the additional constraints and extensions on the UK Core Observation Vital Signs profile" />
+  <description value="Defines the additional constraints and extensions on the UK Core Observation Vital Signs profile, in order to record the clinical observation of a respiratory rate of a patient." />
   <purpose value="To provide implementers with additional support when implementing respiratory rate monitoring and to provide a consistent structure to how the data is presented." />
```

Comment:
I thought we were going to add what is in scope & out of scope for this profile. iirc All respiration, so unaided, Oxygen Concentrators, Nasal Cannula, CPAP Devices, Endotracheal Tube, & Mechanical Ventilator are within this scope.